io/disk/multipath_test.py

```diff
@@ -140,7 +140,6 @@ def tearDown(self):
 
         # Need to wait for some time to make sure multipaths are loaded.
         time.sleep(5)
```

Comment:
is this sleep can also be removed as we are not flushing multipath device

Reply:
@PraveenPenguin No, this sleep is needed still. since we need to wait for multipath to be stable before test exits.

Reply:
aah ok , thanks